librz/util/zip.c

```diff
@@ -107,4 +108,83 @@ RZ_API ut8 *rz_inflatew(const ut8 *src, int srcLen, int *srcConsumed, int *dstLe
 	inflateEnd(&stream);
 	free(dst);
 	return NULL;
-}
\ No newline at end of file
+}
+
+/**
+ * \brief deflate uncompressed data to zlib or gzipped, use MAX_WBITS as the window size logarithm.
+ * \see rz_deflatew()
+ */
+RZ_API ut8 *rz_deflate(RZ_NONNULL const ut8 *src, int srcLen, int *srcConsumed, int *dstLen) {
+	return rz_deflatew(src, srcLen, srcConsumed, dstLen, MAX_WBITS + 32);
```

Comment:
And here

Reply:
But wouldn't that be redundant, since the non-null check is present in `rz_deflatew(...)` (same reasoning as above)?